grammar/grammar.y

```diff
@@ -7,18 +7,17 @@ ContainerMembers
      / KEYWORD_pub? TopLevelDecl ContainerMembers
      / ContainerField COMMA ContainerMembers
      / ContainerField
-     /
 
 TestDecl <- KEYWORD_test STRINGLITERALSINGLE Block
 
 TopLevelComptime <- KEYWORD_comptime BlockExpr
 
 TopLevelDecl
-    <- (KEYWORD_export / KEYWORD_extern STRINGLITERALSINGLE? / (KEYWORD_inline / KEYWORD_noinline))? FnProto (SEMICOLON / Block)
```

Comment:
`noinline` hasn't been removed?

Reply:
So https://ziglang.org/documentation/master/#Grammar is incorrect?

Reply:
The actual parser is currently the only always up-to-date reference for the grammar.

Reply:
Ok, I'll improve it.